features/upgrade/sdn/service-upgrade.feature

```diff
@@ -103,7 +103,7 @@ Feature: service upgrade scenarios
       | name=test-pods |
     And evaluation of `pod.node_name` is stored in the :hostip clipboard
     When I execute on the pod:
-      | curl | <%= cb.hostip %>:<%= cb.port %> |
+      | curl | --connect-timeout | 30 | <%= cb.hostip %>:<%= cb.port %> |
```

Comment:
It seems to me the default connect-time out value is larger than 30s. I'm a little doubt if this change can help. 
Maybe we can add loop here to make it more stable.

Reply:
will --retry option help?